platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -60,6 +65,47 @@ public static X509Certificate fetchGossipCaCertificate(@NonNull final RosterEntr
         return CryptoStatic.decodeCertificate(entry.gossipCaCertificate().toByteArray());
     }
 
+    /**
+     * Fetch a hostname (or a string with an IPv4 address) of a ServiceEndpoint
+     * at a given index in a given RosterEntry.
+     *
+     * @param entry a RosterEntry
+     * @param index an index of the ServiceEndpoint
+     * @return a string with a hostname or ip address
+     */
+    public static String fetchHostname(@NonNull final RosterEntry entry, final int index) {
```

Comment:
I think we should add tests for these util methods

Reply:
I added tests.